### PR TITLE
docs: v11 size token updates for search component

### PR DESCRIPTION
**Pull request by thyhmdo**
This PR updates the size tokens on the Style tab for the search component.

### CODE REVIEW

File: src/pages/components/search/style.mdx [hunk 1]
```diff
@@ -62,12 +62,22 @@ content.
...
 | `.bx--search-magnifier` <br/> `.bx--search-close` | height, width               | 16 / 1   | –             |
 | `.bx--search--lg`                                 | padding-left, padding-right | 48 / 3   | `$spacing-09` |
 | `.bx--search--sm`                                 | padding-left, padding-right | 32 / 2   | `$spacing-07` |
 
+## Size
+
+The width of the search field should appropriately fit the design and layout of
+content.
+
+| Class                                             | Height(px / rem)                    |
```
Comment (alisonjoseph) [inline src/pages/components/search/style.mdx:74]:
```suggestion
| Size                                             | Height(px / rem)                    | 
```

Should we change this to size to be consistent with other components?

**Comment by thyhmdo** [comment]
Unsure if Extra large is an additional size for search

File: src/pages/components/search/style.mdx [hunk 2]
```diff
@@ -62,12 +62,22 @@ content.
...
 
+## Size
+
+The width of the search field should appropriately fit the design and layout of
+content.
+
+| Class                                             | Height(px / rem)                    | 
+| ------------------------------------------------- | ----------------------------------- | 
+| Small (sm)                                        | 32 / 2                              |    
+| Large (lg)                                        | 48 / 3                              |
```
Comment (laurenmrice) [inline src/pages/components/search/style.mdx:77]: - There needs to be a Medium (md) 40 / 2.5 size. There is no Extra Large (xl) size.
- There needs to be a space between Height and (px / rem).

**Review by tay1orjones — APPROVED** [review]

File: src/pages/components/search/style.mdx [hunk 3]
```diff
@@ -62,12 +62,22 @@ content.
...
 | `.bx--search-magnifier` <br/> `.bx--search-close` | height, width               | 16 / 1   | –             |
 | `.bx--search--lg`                                 | padding-left, padding-right | 48 / 3   | `$spacing-09` |
 | `.bx--search--sm`                                 | padding-left, padding-right | 32 / 2   | `$spacing-07` |
 
+## Size
+
+The width of the search field should appropriately fit the design and layout of
+content.
+
+| Size                                              | Height (px / rem)                   |
```
Comment (laurenmrice) [inline src/pages/components/search/style.mdx:74]: - Add a column before the size column called "Element". The first row under the Element column can say "Input".

Comment (laurenmrice) [reply src/pages/components/search/style.mdx:74]: - Also make sure the heading for the data table size section says "Sizes" instead of "Size"

**Review by laurenmrice — APPROVED** [review]
Looks good! ✅

**Pull request merged**